tpcc/worker.go

```diff
@@ -175,7 +175,14 @@ func (w *worker) run(errCh chan<- error, warehouseID int) {
 		if *noWait {
 			warehouseID = rand.Intn(*warehouses)
 		} else {
-			time.Sleep(time.Duration(t.keyingTime) * time.Second)
+			sleepTime := float64(t.keyingTime * 1000)
+			if firstRun {
+				// Sleep for a random duration up to the keying time to smooth out any
+				// potential thundering herd effects when the load generator starts.
+				sleepTime = sleepTime * rand.Float64()
```

Comment:
Thought: You could use the worker index to smoothly deterministically spread over the ramp time. What you have here (randomness) is smoother than what we have right now, but it's not deterministically smooth, and will cause you frustration when you look at experiment after experiment, and each one takes a varying amount of time to settle down.

Reply:
That's what I had at first. Peter suggested replacing that with this simpler method, which prevents the proximal issue of x-treme contention right at the start of the test due to the fact that the workers were all starting in sequence.

Reply:
Gah. Reviews in github over rebases are hard. I should have checked. Apologies.

I disagree with Peter, but by waiting over the long weekend I've given up my right to complain. :)

:shipit:!